packages/plugin-hardhat/src/force-import.ts

```diff
@@ -108,5 +110,14 @@ async function addAdminToManifest(
   opts: ForceImportOptions,
 ) {
   const adminAddress = await getAdminAddress(provider, proxyAddress);
+  if (isEmptySlot(adminAddress)) {
+    // Asser that the admin slot of a transparent proxy is not zero, otherwise the simulation below would fail due to no code at the address.
```

Comment:
```suggestion
    // Assert that the admin slot of a transparent proxy is not zero, otherwise the simulation below would fail due to no code at the address.
```